views/layout.erb

```diff
@@ -33,7 +33,7 @@
       <ul class="group_tabs"></ul>
 
       <div id="content">
-        <%= formatted_file_list("All Files", result.source_files) unless view_gems? %>
+        <%= formatted_file_list("All Files", result, result.source_files) unless view_gems? %>
```

Comment:
Minor point but is an opportunity to cut down the number of method params here by invoking `result.source_files` within `formatted_file_list` method?

```erb
<%= formatted_file_list("All Files", result) unless view_gems?
```

Reply:
good call cleaned this up